js_modules/dagster-ui/packages/ui-core/src/asset-graph/sidebar/Node.tsx

```diff
@@ -345,6 +350,7 @@ const ExpandMore = styled.div``;
 const GrayOnHoverBox = styled(Box)`
   border-radius: 8px;
   cursor: pointer;
+  user-select: none;
```

Comment:
guessing this is to stop the double click from highlighting the text, a little unfortunate but I kinda do want to still allow users to highlight the text 😬

Reply:
yeah, I'm not sure what to do about this one -- I think it's a bit odd to select text inside of items that are "selectable" in the UI, sort of a blue highlight inside a blue highlight. I think it became more of a problem in this PR because I added double-click-to-expand... maybe we can improve this as part of your focus / keyboard stuff